sdk/src/RaygunClient.ts

```diff
@@ -102,7 +102,7 @@ const init = (raygunClientOptions: RaygunClientOptions) => {
     }
 
     initialized = true;
-    RaygunLogger.i(`RaygunClient version initialized`)
+    RaygunLogger.d(`RaygunClient version initialized`)
```

Comment:
Is the actual 'version' able to printed? E.g. 'RaygunClient v1.0.0 initialized'

Reply:
In the RaygunLogger, I have implemented it such that for every "version" it will be replaced with the version number... didn't want the logic to be scattered throughout the project. I can make the key for version more obscure if its not obvious enough